.travis.yml

```diff
@@ -31,9 +26,27 @@ addons: &addons
         #      - tcc
         # - tcc:i386
 
-#matrix:
-#  include:
-#    - compiler: gcc
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+      compiler: clang
+      script:
+        - if [ -n "$COVERITY_SCAN_TOKEN" ]; then exit 0; fi
+        - ./autogen.sh $TESTS
+        - ./configure $TESTS
+        - make
+        - if [ -n "$TESTS" ]; then ./udffsck/travis-tests.sh $TESTLEVEL; fi
+    - os: osx
+      compiler: clang
+      script:
+        - if [ -n "$COVERITY_SCAN_TOKEN" ]; then exit 0; fi
+        - ./autogen.sh $TESTS
+        - ./configure $TESTS
+        - cd libudffs && make
+        - cd ../udffsck && make
```

Comment:
Why is testing, x86/x86_64 and gcc disabled for osx? All those tests shall be run.

Keeping osx as allowed failure is good idea, since it is primarily linux package.